volume/store/store.go

```diff
@@ -247,15 +249,17 @@ func (s *VolumeStore) CreateWithRef(name, driverName, ref string, opts, labels m
 
 // Create creates a volume with the given name and driver.
 // This is just like CreateWithRef() except we don't store a reference while holding the lock.
-func (s *VolumeStore) Create(name, driverName string, opts, labels map[string]string) (volume.Volume, error) {
-	return s.CreateWithRef(name, driverName, "", opts, labels)
+// In case a volume with the name has already been created, an errAlreadyCreated error will be
+// returned along with the already created volume, if `quiet` is not true
+func (s *VolumeStore) Create(name, driverName string, opts, labels map[string]string, quiet bool) (volume.Volume, error) {
```

Comment:
Thanks @cpuguy83. No sure about the best way to handle the case. Any suggestions?